src/main/scala/mesosphere/marathon/core/instance/Goal.scala

```diff
@@ -0,0 +1,54 @@
+package mesosphere.marathon
+package core.instance
+
+import play.api.libs.json.{Format, JsError, JsResult, JsString, JsSuccess, JsValue, JsonValidationError, Reads, Writes}
+
+/**
+  * Defines goal of the instance.
+  * Goal is set by an orchestration layer and interpreted by scheduler layer.
+  * In the end it is used by low-level scheduler to make scheduling decisions e.g. should the task associated with this instance be launched or killed?
+  */
+sealed trait Goal extends Product with Serializable
+
+object Goal {
+
+  /**
+    * There should always be a running Mesos task associated by instance in this state.
+    * Instance with Suspended Goal might be changed to both [[Running]] or [[Stopped]] by orchestration layer.
+    */
+  case object Running extends Goal
+
+  /**
+    * Tasks associated with this instance shall be killed, but the instance needs to be kept in the state.
```

Comment:
This could follow the above suggestion:

_Expresses the intent that tasks associated with this instance shall be killed, but the instance needs to be kept in the state._